foodsaving/groups/serializers.py

```diff
@@ -108,6 +119,64 @@ def create(self, validated_data):
         return group
 
 
+class AgreementSerializer(serializers.ModelSerializer):
```

Comment:
Shouldn't this also return the users that have agreed? Can be shown in frontend to the agreement manager.

Reply:
Could do, and might be useful at some point, I just implemented the "minimum information" principle for now...

Another approach would be to include it in the membership objects in the group serializer (for membership managers), as that is where other membership management information is (e.g. roles).

Reply:
Yes, that seems to be easy enough to get added later when the frontend is in development or when a user requests it. It's fine to progress without this information for now.